src/styles/Flipper.module.scss

```diff
@@ -0,0 +1,37 @@
+.flipperWrapper {
+    display: flex;
+    position: relative;
+    align-items: center;
+    justify-content: center;
```

Comment:
I think the aspect-ratio css style might be needed here, when I tried `iPad Air` in the inspector for instance, it got weird:
![image](https://user-images.githubusercontent.com/6663002/227783360-87a33ff5-ce55-43d4-92b1-e50b79470f0f.png)

Reply:
found the problem, it was a misplaced height setting for smaller screens.